docs/shared/plugin-features/use-task-executors.md

```diff
@@ -83,3 +83,23 @@ Or you can overwrite individual executor options like this:
 nx [command] [project] --[optionNameInCamelCase]=[value]
 nx build cart --outputPath=some/other/path
 ```
+
+## nx:run-commands shorthand
```

Comment:
Document this for a reader with the point of view of wanting to run a shell command rather than "wanting to get a short hand for run-commands"

Reply:
+1

Reply:
I think changing the heading here is actually enough to change that POV, I'm going to merge w/ updated heading. Let me know if you feel it needs follow up.